lib/faulty/storage/redis.rb

```diff
@@ -348,7 +348,7 @@ def all_list_keys
         num_blocks = (options.circuit_ttl.to_f / options.list_granularity).floor + 1
         start_block = current_list_block - num_blocks + 1
         num_blocks.times.map do |i|
-          key('list', start_block + i)
+          key('list', "{#{start_block + i}}")
```

Comment:
I wonder if it would be worth adding a private `#hashtag` method to make the intention and readability of these interpolations a bit better 🤔

```ruby
def hashtag(str)
  "{#{str}}"
end
```
e.g.

```suggestion
          key('list', hashtag(start_block + i))
```

Maybe not 🤷🏼‍♂️

Reply:
Ah good idea! I'll action.

Reply:
https://github.com/buildkite/faulty/pull/1/commits/4c49eaee9e81440df8e2ea480baf172458709dca